api/server.py

```diff
@@ -25,12 +25,16 @@ def home():
     # Add a default stampified page here. For now using placeholder story.
     url = ""
     max_pages = "4"
+    animations = False
     if 'website_url' in request.args:
         url = request.args.get('website_url')
     if 'max_pages' in request.args:
         max_pages = request.args.get('max_pages')
+    if 'animations' in request.args:
+        animations = request.args.get('animations')
 
-    stampified_url = '/stampified_url?url=%s&max_pages=%s' % (url, max_pages)\
+    stampified_url = '/stampified_url?url=%s&max_pages=%s&animations=%s' % \
```

Comment:
Use [urllib](https://docs.python.org/3/library/urllib.parse.html) to build URLs.

Reply:
Thanks for the suggestion! Updated!

Reply:
https://stackoverflow.com/questions/7801991/how-to-convert-a-dictionary-to-query-string-in-python

Reply:
Will look into it. Missed this earlier. Thanks for the link.